qemu/tests/usb_storage.py

```diff
@@ -148,9 +135,6 @@ def _check_serial_option(serial, regex_str, expect_str):
         if serial not in ["EMPTY_STRING", "NO_EQUAL_STRING"]:
             # Verify in guest when serial is set to empty/null is meaningless.
             _verify_string(serial, output, [serial])
-        _check_and_umount_usb(session)
-        _do_io_test_guest(session)
```

Comment:
remove i/o test for check_serial_option, check_removable_option and check_io_size_option, so I suggest you add them to commits

Reply:
Add the corresponding info to the comments. Thanks!